ADOL-C/src/drivers/psdrivers.c

```diff
@@ -143,7 +143,7 @@ int directional_active_gradient(short tag,      /* trace identifier */
       sum = 0;
       for(i=0;i<s;i++)
         {
-          sum += fabs(sigma_g[i]);
+          sum += fabs((double)sigma_g[i]);
```

Comment:
I think it's better to simply use abs here

Reply:
Thanks! I changed it